blocks/search-results/search-results.js

```diff
@@ -72,11 +72,13 @@ async function searchPages(placeholders, term, page) {
   const curPage = result.slice(startResult, startResult + resultsPerPage);
 
   curPage.forEach((line) => {
+    let searchTitle;
     const res = document.createElement('div');
     res.classList.add('search-result');
     const header = document.createElement('h3');
     const link = document.createElement('a');
-    const searchTitle = line.pagename || line.breadcrumbtitle || line.title;
+    if (line.pagename === 'Newsroom') searchTitle = line.breadcrumbtitle;
```

Comment:
There's a `type` field which you can use to know whether its a "Newsroom" or "Health Thinking" page.